enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -96,6 +98,9 @@ func (c *podmanCommandContainer) getPodmanRunArgs(workDir string) []string {
 	if c.options.CapAdd != "" {
 		args = append(args, "--cap-add="+c.options.CapAdd)
 	}
+	for _, device := range c.options.Devices {
+		args = append(args, "--device="+device.PathOnHost)
```

Comment:
I think podman (like docker) uses the `hostpath:containerpath:perms` flag format -- something like this should probably work?

```suggestion
		deviceFlag := device.PathOnHost
		if device.PathInContainer != "" {
				deviceFlag += ":" + device.PathInContainer
				if device.CgroupPermissions != "" {
						deviceFlag += ":" + device.CgroupPermissions
				}
		}
		args = append(args, "--device="+deviceFlag)
```